identity-account/Cargo.toml

```diff
@@ -34,17 +34,17 @@ tokio = { version = "1.5", features = ["sync"] }
 zeroize = { version = "1.4" }
 
 [dependencies.iota-crypto]
-version = "0.5"
+version = "0.7"
 features = ["blake2b", "ed25519", "hmac", "pbkdf", "sha", "slip10", "std"]
 
 [dependencies.iota_stronghold]
 git = "https://github.com/iotaledger/stronghold.rs"
-rev = "6dd92dc9743eba2f3b4126425e7572470d92c80b"
+rev = "ad57181e7c5baa4b6ccb66fb464667c97967db08"
```

Comment:
it looks like that this was the latest revision. is there a specific reason to not use head?

Reply:
You mean to specify `branch = "dev"` rather than a specific commit? We've just had some bad experience with `iota-client` not pinning their commits to a specific version, which broke our build randomly. We'd thus rather do a manual upgrade :slightly_smiling_face:

Reply:
ok. if stronghold reaches stable, I would advise using the master branch.

Reply:
I think that this is debatable, because it is always safer to pin to a rev (even if it is a bit more opaque as to which rev you are actually consuming.) The other point is that there is only a `main` branch, not a `master` branch.